aea/protocols/base.py

```diff
@@ -43,6 +43,26 @@
 Address = str
 
 
+class MessageHeader(
+    namedtuple("MessageHeader", ("message_id", "performative", "sender", "target"))
+):
+    """
+    Basic message fields.
+
+    message_id
+    performative
+    sender
```

Comment:
sender does not need to be maintained after dialogue constructed as same for all messages